src/environment/stella_environment.cpp

```diff
@@ -178,6 +178,12 @@ void StellaEnvironment::soft_reset(){
   m_state.incrementFrame();
 }
 
+/** Emulates a NOOP action for both players */
+void StellaEnvironment::wait()
```

Comment:
I would prefer if we didn't have the wait() method, and instead directly applied PLAYER_A_NOOP, PLAYER_B_NOOP where relevant. This makes it clear that what we mean to do is emulate the game for a number of steps. See also StellaEnvironment::reset().